tests/unit/woopay/test-class-woopay-session.php

```diff
@@ -14,6 +14,11 @@
  * WooPay_Session unit tests.
  */
 class WooPay_Session_Test extends WCPAY_UnitTestCase {
+	/**
+	 * @var Database_Cache|MockObject
+	 */
+	protected $mock_cache;
+
```

Comment:
I'm assuming we should be removing this as well? It's not used anywhere in the test file.

Reply:
I actually had just added it as part of reverting the tests as I was getting some lint complaints in the editor. It's used in the https://github.com/Automattic/woocommerce-payments/pull/7055/files/a63d301cb29eb550ae77f352d5d85c46309a9670#diff-e2ba3588802b8c413a54ea81192a62c55e1db5aaa904edf9119c08b4cfa62417R212 in the test.

Reply:
Ah, let's keep it then!